README.md

```diff
@@ -233,6 +238,22 @@ You may be interested in:
 * [demos/io_uring](./demos/io_uring), how ring buffers can eliminate kernel syscalls, reduce server hardware requirements by a factor of two, and change the way we think about event loops.
 * [demos/hash_table](./demos/hash_table), how linear probing compares with cuckoo probing, and what we look for in a hash table that needs to scale to millions (and billions) of account transfers.
 
+## Short-Term Roadmap
+
+Done:
+
+1. Consensus layer readiness
+
+In progress:
+
+2. Storage layer readiness
+  * Until this happens you'll occasionally see panics in the LSM code.
```

Comment:
Would be great just to explain why the panics happen. i.e. Assertion failures as we tune overtight or incorrect assertions, or as assertions detect bugs and shut TigerBeetle down safely.

We don't want people to panic when they see that TigerBeetle panics. Rather, to explain this also to build trust.

Reply:
Over time, assertion failures will become more and more rare, as we increase assurance through testing.

Reply:
Good call, added.